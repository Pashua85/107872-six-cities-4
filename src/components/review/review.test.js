import React from 'react';
import Enzyme, {shallow} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Review from './review';
import places from '../../mock/test-offers';

Enzyme.configure({
  adapter: new Adapter()
});

describe(`Review`, () => {
  it(`should render correctly`, () => {
    const wrapper = shallow(<Review review={places[0].reviews[0]} />);
    expect(wrapper).toMatchSnapshot();
  });
});
