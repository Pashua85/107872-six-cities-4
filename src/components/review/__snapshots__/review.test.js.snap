// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Review should render correctly 1`] = `ShallowWrapper {}`;
